### DISCUSSION TITLE
What is litellm_uuid file used for?

### DISCUSSION THREAD

**Discussion by codeacme17** [BerriAI/litellm#317] [Q&A] [answered]
In my project, a `litellm_uuid.txt` file is generated. I am not sure what this file does. And I want to know if this file needs to be added to `.gitignore`

**Comment by ishaan-jaff** [comment, marked as answer]
@codeacme17 

- we use this for tracking what llms are being used & for error/crash reporting on litellm (I want to clarify we do not log your requests/responses. we just want to know what llms are used and errors thrown so we can improve the litellm usage)

- it's optional if you want to add it to .gitignore, not a requirement. It should be okay to add it to your gitignore

**Reply by codeacme17** [reply]
okay，thanks for your answer I understand now
